Tests/PayID/XRPPayIDIntegrationTests.swift

```diff
@@ -13,8 +13,8 @@ final class PayIDIntegrationTests: XCTestCase {
   func testResolvePaymentPointerKnownPointerMainnet() {
     let expectation = XCTestExpectation(description: "resolveToXRP completion called.")
 
-    // GIVEN a Pay ID that will resolve on Mainnet and a PayID client.
-    let payIDClient = PayIDClient(network: .main)
+    // GIVEN a Pay ID that will resolve on mainnet and a PayID client.
```

Comment:
There's tests in here for PayIDClient as well and you've renamed it to be XRP specific. In other languages I've just left this called 'PayIDIntegrationTests' since it encompasses all sorts of end to end tests related to PayID. 

Would you mind reverting? Sorry as I know moving files around is a pain.

Reply:
No problem - the analogous file in `xpring-js` is called `xrp-pay-id-client-integration-test` so I've made a note to rename that.